plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/AbstractTestDeltaLakeCreateTableStatistics.java

```diff
@@ -437,18 +436,14 @@ public void testMultiFileTable()
             throws Exception
     {
         DeltaLakeColumnHandle columnHandle = new DeltaLakeColumnHandle("name", createUnboundedVarcharType(), "name", createUnboundedVarcharType(), REGULAR);
-        Session session = testSessionBuilder()
-                .setCatalog(DELTA_CATALOG)
-                .setSystemProperty("scale_writers", "false")
-                .setSchema(SCHEMA)
-                .build();
-        try (TestTable table = new TestTable(
-                "test_partitioned_table_",
-                ImmutableList.of(),
-                ImmutableList.of(),
-                "SELECT name FROM tpch.tiny.nation UNION select name from tpch.tiny.customer",
-                session)) {
-            List<AddFileEntry> addFileEntries = getAddFileEntries(table.getName());
+        String tableName = "test_multifile_table_" + randomTableSuffix();
```

Comment:
i must admit i don't know what testMultiFileTable() is for.
Maybe we can remove it, or add a javadoc explaining its value
cc @alexjo2144

Reply:
I may have known at one point, but I'm also unsure if it has any value

Reply:
if we don't know why we have a test, we don't know what's a good change and what isn't for the test prep or assertions.
let's drop it.

Reply:
I dropped the useless test in a separate PR.
`testMultiFileTableWithNaNValue` has been reworked to ensure that the table tested contains multiple active files.